pym/bob/scripts.py

```diff
@@ -178,6 +181,9 @@ def cmd():
             from .input import RecipeSet
             RecipeSet.ignoreCommandCfg()
 
+        if args.color_mode:
+            setColorMode(args.color_mode)
```

Comment:
You could do it like above to set the command line preference on the `RecipeSet` class and do a single `setColorMode()` in `__parse` after the user configuration was read but before the recipes are parsed.